client/me/purchases/vat-info/index.tsx

```diff
@@ -44,11 +46,11 @@ export default function VatInfoPage() {
 			<Column type="sidebar">
 				<Card className="vat-info__sidebar-card">
 					<CardHeading tagName="h1" size={ 16 } isBold={ true } className="vat-info__sidebar-title">
-						{ translate( 'VAT Information' ) }
+						{ translate( 'Business Tax ID Information' ) }
```

Comment:
To save on translations, I think (?) this could probably be `Business Tax ID details` also (or whatever we end up using for the title string).